### ISSUE TITLE
[Bug]: Insufficient color contrast when ProgressIndicator is used in a layer

### ISSUE THREAD

**Issue by daniel-suski** [carbon-design-system/carbon#10110]
### Package

carbon-components, carbon-components-react, @carbon/colors, @carbon/layout, @carbon/themes

### Package version

carbon-components: 10.46.0, carbon-components-react": 7.46.0, @carbon/colors: 10.33.0, @carbon/themes: 10.45.0, @carbon/layout: 10.33.0

### Description

I'm using a ProgressIndicator in a layer in dark mode (Tearsheet component from IBM C&CS PAL v.0.97 with ProgressIndicator in influencer pane) and the disabled steps in the progress indicator are using the wrong grey shade given the slightly lighter background of the Tearsheet influencer pane, resulting in an insufficient color contrast.
I was looking for a "light=true" property like it exists for other components (e.g. TextInput), but could not find one.

Mentioning @dcwarwick who is aware of the issue, in case something needs to be modified in the C&CS PAL.

Please ping me via slack@ibm if a screenshot showing the contrast issue would help.

This issue could be related to https://github.com/carbon-design-system/carbon/issues/9877 but I'm not sure.

### CodeSandbox example

n/a

### Steps to reproduce

n/a

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by jnm2377** [comment]
Hi @daniel-suski thanks for opening this issue. Could you provide a codesandbox recreating the issue? Thanks!

**Comment by abbeyhrt** [comment]
This will be addressed in Carbon v11 but we will see about a workaround for this issue in v10!

**Comment by tay1orjones** [comment]
IIUC this was resolved by https://github.com/carbon-design-system/carbon/pull/10992
